omod/src/main/webapp/pages/registerPatient.gsp

```diff
@@ -299,11 +300,10 @@ fieldset[id\$="-fieldset"] div > div {
 
                             ${combineSubSections == true ? "" : "<legend id=\"birthdateLabel\">" + ui.message("registrationapp.patient.birthdate.label") + "</legend>"}
 
-                            <h3>${ui.message("registrationapp.patient.birthdate.question")}</h3>
-                            <div style="color: #999;">(${ ui.message("emr.formValidation.messages.requiredField.label") })</div>
                             ${ ui.includeFragment("uicommons", "field/multipleInputDate", [
-                                    label: "",
+                                    label: ui.message("registrationapp.patient.birthdate.question"),
                                     formFieldName: "birthdate",
+                                    classes: ["requiredTitle"],
```

Comment:
Why the addition of the new class "requiredTitle" here?

Reply:
Because we want add 'required' label only to birthdate title label i.e. What's the patient's birth date? (required), not to each field in birthdate section. If we pass 'required' class to multipleInputDate fragment, all fragments which are in multipleInputDate will have 'required' class property and will display '(required)' label next to each label e.g. Day(required) Month(required) Year(required) Estimated Years(required) Estimated Months(required). That's why I've added this 'requiredTitle' class - to add 'required' label only in title label, not to each field. And I used this 'requiredTitle' class in https://github.com/openmrs/openmrs-module-uicommons/pull/82

Reply:
Thanks @druchniewicz !  Is there no way we can make the multipleDateInput widget smart enough to so that when you flag it as "required" it only means to add the label once?  If there's something in the way we did things that makes this difficult, no worries then, I'll go ahead and approve this... I just think requiredTitle is slighty unintuitive.

Reply:
I think it would be very difficult to refactor it, because multipleInputDate fragment renders another fragments e.g. dropDown or text fragments and if we would add 'required' property class, this 'required' label would be visible in each place in system where those fragments are used (what is not expected behaviour). That's why adding the separate class property for this field is the fastest and safest solution and doesn't require much changes in code.

Reply:
Thanks, sounds good!